backend/migrations/Version20260828160000.php

```
@@ -0,0 +1,64 @@
+            LEFT JOIN comic c ON c.id = r.linked_comic_id
+            SET r.linked_user_id_snapshot = r.linked_user_id,
+                r.linked_comic_id_snapshot = r.linked_comic_id,
+                r.linked_share_id_snapshot = r.linked_share_id,
+                r.linked_comic_title_snapshot = c.title
+            WHERE r.linked_user_id IS NOT NULL
+               OR r.linked_comic_id IS NOT NULL
+               OR r.linked_share_id IS NOT NULL
+        SQL);
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)